fast64_internal/f3d/f3d_material.py

```diff
@@ -2588,12 +2594,10 @@ def draw(self, context):
         ui_other(world.rdp_defaults, world, layout, True)
 
 
-def getOptimalFormat(tex, useLargeTextures, isMultitexture):
+def getOptimalFormat(tex, curFormat, isMultitexture):
     texFormat = "RGBA16"
-    if useLargeTextures:
-        return "RGBA16"
-    if bpy.context.scene.ignoreTextureRestrictions or tex.size[0] * tex.size[1] > 8192:  # Image too big
-        return "RGBA32"
+    if (tex.size[0] * tex.size[1] > 8192) or isMultitexture:  # Image too big
```

Comment:
```suggestion
    if isMultitexture:
        return curFormat
    if (tex.size[0] * tex.size[1] > 8192):  # Image too big
```
just because it's weird the "too big" comment seems to apply to multitextured too

or the `isMultitexture` check could be moved to where `getOptimalFormat` is called